okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/OkHttpClients.java

```diff
@@ -44,6 +47,12 @@
      */
     private static final ExecutorService executorService = Tracers.wrap(new Dispatcher().executorService());
 
+    /**
+     * The {@link ScheduledExecutorService} used for scheduling call retries.
+     */
+    private static final ScheduledExecutorService scheduledExecutorService =
+            Tracers.wrap(new ScheduledThreadPoolExecutor(0, Util.threadFactory("OkHttp Scheduler", false)));
```

Comment:
we should indicate that this is a remoting threadpool and not an OkHttp threadpool.

Reply:
done